utility/utils.js

```diff
@@ -388,7 +388,7 @@ class Utils {
    */
   static async updatePugStats(match_id, map_id, team1_id, team2_id, winner, deleteTeams = true) {
     let teamAuthSql =
-      "SELECT GROUP_CONCAT(CONCAT('\"', ta.auth, '\"')) as auth_name, GROUP_CONCAT(CONCAT('\"', ta.name, '\"')) as name FROM team_auth_names ta WHERE team_id = ?";
+      "SELECT GROUP_CONCAT(ta.auth) as auth_name, GROUP_CONCAT(CONCAT('\"', ta.name, '\"')) as name FROM team_auth_names ta WHERE team_id = ?";
```

Comment:
Uhh changing this may have further consequences in match creation other than pugging, why exactly were the quotes removed from the Steam IDs? Last I checked these need to be there as the steam ID value is too large to be represented as an int for sourcemod.

Reply:
I'm not sure why it's added? isn't steam_id string in the database?
when keeping quotes I get error
```
Error: Data too long for column 'steam_id' at row 1
    at PromisePool.query (/root/G5API/node_modules/mysql2/promise.js:341:22)
    at Database.query (file:///root/G5API/db.js:23:31)
    at Function.updatePugStats (file:///root/G5API/utility/utils.js:462:16)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async file:///root/G5API/routes/matches/matchserver.js:220:11 {
  code: 'ER_DATA_TOO_LONG',
  errno: 1406,
  sql: 'INSERT INTO player_stats SET `match_id` = \'100\', `map_id` = 72, `team_name` = \'someone▒^=▒▒\', `steam_id` = \'\\"76561197989783553\\"\', `name` = \'\\"someone▒^=▒▒\\"\', `winner` = 0',
  sqlState: '22001',
  sqlMessage: "Data too long for column 'steam_id' at row 1"
  ```

Reply:
Oh you know what, I think that's okay. After splitting the pug stats I think that may have been a mistake, As long as its working for you I trust you :)